json_rpc/server.nim

```diff
@@ -1,10 +1,10 @@
 import
   std/tables,
-  chronos,
+  faststreams/async_backend,
```

Comment:
why would fs be needed here?

Reply:
to support the ability to compile in chronos/asyncdispatch mode(i. e. providing async)

Reply:
but faststreams is not used in this module? perhaps you need an export somewhere else

Reply:
faststream/async_backend is used only to export conditionally `chronos` or `asyncdispatch` stuff based on `asyncBackend`.

Reply:
right - so with `asyncBackend=none` (the default), nothing will be exported by `async_backend` - this is how we use the library generally in projects.